devdoc/constbuffer_requirements.md

```diff
@@ -340,6 +372,8 @@ CONSTBUFFER_FROM_BUFFER_RESULT CONSTBUFFER_from_buffer(const unsigned char* sour
 
 **SRS_CONSTBUFFER_02_065: [** If `destination` is `NULL` then `CONSTBUFFER_from_buffer` shall fail and return `CONSTBUFFER_FROM_BUFFER_RESULT_INVALID_ARG`. **]**
 
+**SRS_CONSTBUFFER_51_027: [** If state of `destination` is `CONSTBUFFER_WRITABLE_STATE_OPEN`, shall fail and return `CONSTBUFFER_FROM_BUFFER_RESULT_INVALID_ARG`. **]**
```

Comment:
destination is the out argument, this function is actually creating the buffer placed in destination, so we should definitely not look at it.

Reply:
Got it. Fixed.